src/libstd/thread/local.rs

```diff
@@ -18,7 +18,6 @@ use cell::UnsafeCell;
 
 // Sure wish we had macro hygiene, no?
 #[doc(hidden)]
-#[unstable(feature = "thread_local_internals")]
```

Comment:
The module at the top-level has an `#[unstable]` attribute which everything inside inherits from by default. (it was intentional)